rad/src/reducers/mod.rs

```diff
@@ -10,17 +10,20 @@ use witnet_data_structures::radon_report::ReportContext;
 
 pub mod average;
 pub mod deviation;
+pub mod hash_concatenate;
 pub mod median;
 pub mod mode;
 
-#[derive(Debug, PartialEq, TryFromPrimitive)]
+#[derive(Debug, PartialEq, TryFromPrimitive, IntoPrimitive)]
 #[repr(u8)]
 pub enum RadonReducers {
     // Implemented
     Mode = 0x02,
     AverageMean = 0x03,
     AverageMedian = 0x05,
     DeviationStandard = 0x07,
+    HashConcatenate = 0x0b,
+    Unwrap = 0x0c,
```

Comment:
Interesting. Where did this come from?

Reply:
We dont have a reducer to convert an array of 1 element in that element

Reply:
Deleted unwrap, using Mode instead of